contrib/golang/upstreams/http/tcp/test/test_data/http2dubbo-example/README.md

```diff
@@ -0,0 +1,25 @@
+# Documentation for http2dubbo by golang extension
```

Comment:
please also add an example that include a HTTP request and response, with a demo dubbo service.
i.e. a curl command example.

Reply:
done. in the dir: /envoy/contrib/golang/upstreams/http/tcp/test/test_data/http2dubbo-example .